Client/Frontend/UIConstants.swift

```diff
@@ -5,6 +5,12 @@
 import Foundation
 import Shared
 
+public extension UIColor {
+    class func toolbarTintColor() -> UIColor {
```

Comment:
Not sure I like having an an extension on UIColor here for just this one property, it seems to go against the conventions we have been using for colour definitions everywhere else. I actually like the idea of having UIColor extensions to define colours, but dislike inconsistency more.

Also, if this is the same toolbar tint colour as used elsewhere in the app (the browser toolbar, menu etc) then we should update those elements to use the same function to ensure that if we change the toolbar tint for a privacy mode we change it for everything that uses it.

Also, this only returns the toolbarTintColor for non-private mode, so the name should either reflect that or the function should take an `isPrivate` arg to determine the right toolbar tint colour.